.github/workflows/antithesis-test.yml

```diff
@@ -0,0 +1,112 @@
+---
+name: Build and trigger Antithesis exploration
+
+on:
+  push:
```

Comment:
Remove.

Reply:
OK, so we are only doing nightly then! Got it!